src/components/SearchInput.js

```diff
@@ -44,6 +45,7 @@ class SearchInput extends React.Component {
           focusOnLoad={this.props.focusOnLoad}
           handleResetClick={this.props.handleResetClick}
           icon='search'
+          elementRef={this.props.elementRef}
```

Comment:
Sorry, it looks like the CI build is failing now because this line is not alphabetized